logistics_project/apps/malawi/warehouse/report_views/stock_status.py

```diff
@@ -1,9 +1,14 @@
 from __future__ import unicode_literals
 from builtins import range
 import json
+import folium
+import logging
+import random
 from collections import defaultdict
+from geopy.geocoders import Nominatim
```

Comment:
also likely this one?

Reply:
Yes it will be included in the requirements files